SQL/9999-99-99-drop_tables.sql

```diff
@@ -179,3 +179,5 @@ DROP TABLE IF EXISTS `hrrt_archive_files`;
 DROP TABLE IF EXISTS `hrrt_archive`;
 DROP TABLE IF EXISTS `Project`;
 DROP TABLE IF EXISTS `subproject`;
+DROP TABLE IF EXISTS `appointment_type`;
+DROP TABLE IF EXISTS `appointment`;
```

Comment:
The positioning here would not work due to the foreign key on the session table. this drop needs to occur before the session drop